.github/workflows/ci-rosetta.yaml

```diff
@@ -0,0 +1,88 @@
+name: release harmony
+
+on:
+  push:
+    branches: ['rosetta_test']
```

Comment:
is it the right branch?

Reply:
this ci is only for test. will remove after all things done